lib/index.js

```diff
@@ -34,6 +34,12 @@ module.exports = function reshapeStandard (options = {}) {
   if (!contentOpt.md) {
     const markdownOpt = { typographer: true, linkify: true }
     const md = new MarkdownIt(Object.assign({}, markdownOpt, options.markdown))
+    // if the user has provided markdown-it plugins, load them into the parser instance
+    if (options.markdownPlugin) {
+      for (let prop in options.markdownPlugin) {
+        md.use(options.markdownPlugin[prop])
+      }
+    }
```

Comment:
You can also run this with a `map` for a slightly cleaner impmentation:

```js
(options.markdownPlugin || []).map((plugin) => { md.use(plugin) })
```

This implementation would also allow you to cut the if statement wrapper